docs/docs/configuration/camera_specific.md

```diff
@@ -231,3 +231,40 @@ ffmpeg:
 ### TP-Link VIGI Cameras
 
 TP-Link VIGI cameras need some adjustments to the main stream settings on the camera itself to avoid issues. The stream needs to be configured as `H264` with `Smart Coding` set to `off`. Without these settings you may have problems when trying to watch recorded footage. For example Firefox will stop playback after a few seconds and show the following error message: `The media playback was aborted due to a corruption problem or because the media used features your browser did not support.`.
+
+## USB Cameras (aka Webcams)
+
+In some scenarios one may desire using a USB Camera (i.e. webcam) with Frigate. The example instructions below walk through setting one up with go2rtc's [FFmpeg Device](https://github.com/AlexxIT/go2rtc?tab=readme-ov-file#source-ffmpeg-device) support:
+
+- Preparation outside of Frigate:
+  - Get USB camera path. Run `v4l2-ctl --list-devices` to get a listing of locally-connected cameras available. (You may need to install `v4l-utils` in a way appropriate for your Linux distribution). In the sample configuration below, we use `video=0` to correlate with a detected device path of `/dev/video0`
+  - Get USB camera formats & resolutions. Run `ffmpeg -f v4l2 -list_formats all -i /dev/video0` to get an idea of what formats and resolutions the USB Camera supports. In the sample configuration below, we use a width of 1024 and height of 576 in the stream and detection settings based on what was reported back.
+  - If using Frigate in a container (e.g. Docker on TrueNAS), ensure you have USB Passthrough support enabled, along with a specific Host Device (`/dev/video0`) + Container Device (`/dev/video0`) listed.
+
+- In your Frigate Configuration File, add the go2rtc stream and roles as appropriate:
+
+```
+go2rtc:
+  streams:
+    usb_camera:
+      - "ffmpeg:device?video=0&video_size=1024x576#video=h264" 
+
+cameras:
+  usb_camera:
+    enabled: true
+    ffmpeg:
+      inputs:
+        - path: rtsp://127.0.0.1:8554/usb_camera
+          input_args: preset-rtsp-restream
+          roles:
+            - detect
+            - record
+    detect:
+      enabled: false # <---- disable detection until you have a working camera feed
+      width: 1024
+      height: 576
+```
+
+There may be additional ffmpeg device arguments depending on your camera, you may want to check out issues [1184](https://github.com/blakeblackshear/frigate/issues/1184), [3213](https://github.com/blakeblackshear/frigate/issues/3213), and [4062](https://github.com/blakeblackshear/frigate/issues/4062) for more information. Note that these discussions may contain instructions before go2rtc support was added to Frigate and will differ in configuration setup.
```

Comment:
Not sure if we want to include this in the official documentation as they are pretty old, and especially since they contain things that may not apply anymore. Users can always search the past issues and discussions if they need more assistance.

Reply:
Removed, thanks again.